EventView-1/src/main/java/com/eventview/model/EmailTemplate.java

```diff
@@ -0,0 +1,43 @@
+package com.eventview.model;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.Max;
+
+public class EmailTemplate {
+    @Email
```

Comment:
in future u use velocity email templates or mustache templates for now this is ok.